pkg/apply/filter/dependency-filter_test.go

```diff
@@ -72,10 +67,10 @@ func TestDependencyFilter(t *testing.T) {
 				taskContext.InventoryManager().AddPendingApply(idA)
 				taskContext.AddInvalidObject(idInvalid)
 			},
-			id:               idA,
-			expectedFiltered: true,
-			expectedReason:   fmt.Sprintf("dependency invalid: %q", idInvalid),
-			expectedError:    nil,
+			id: idA,
+			expectedError: testutil.EqualError(
```

Comment:
`assert.Equal` can't handle comparing `fmt.Errorf`, because it doesn't implement the `Is` or `Equal` methods. So we have to wrap it with `testutil.EqualError` to use the type and error string for comparison instead.